### ISSUE TITLE
running `bin/nur update` is too expensive

### ISSUE THREAD

**Issue by milahu** [nix-community/NUR#351]
the readme says

> How to add your own repository.
>
> At the moment each URL must point to a git repository. By running `bin/nur update` the corresponding `repos.json.lock` is updated and the repository is  tested. This will perform also an evaluation check, which must be passed for your repository. Commit the changed `repos.json` but NOT `repos.json.lock`

so i add my repo, run `bin/nur update` ... and my whole computer hangs ^^

lets ignore the facts that i should enable swap and use ulimit:
`bin/nur update` should not check all repos, but only the one repo (or few repos) that i want to add

the added repo can be found with `git diff origin/HEAD`
or `diff <(curl https://raw.githubusercontent.com/nix-community/NUR/master/repos.json) repos.json`

**Comment by Mic92** [comment]
Right, when I started the project this was never an issue :) Now NUR grew quite a bit.

**Comment by Mic92** [comment]
But actually we start a new nix-env for every repo. So maybe its only a few repos that consumes a lot of ram.

**Comment by Pandapip1** [comment]
Solved by no longer eval-ing every repo
